src/grt/src/GlobalRouter.tcl

```diff
@@ -420,9 +420,9 @@ proc global_route_debug { args } {
   set tree2D [info exists flags(-tree2D)]
   set tree3D [info exists flags(-tree3D)]
   set db_block [ord::get_db_block]
-  set net [$db_block findNet $keys(-net)]
 
   if { [info exists keys(-net)] && $net != "NULL" } {
```

Comment:
you are setting net after checking if it is NULL

Reply:
Thanks! Fixed in the last commit.